setup.py

```diff
@@ -104,9 +104,9 @@
         entry_points={
             "console_scripts": ["trl=trl.commands.cli:main"],
         },
-        package_data={"trl": ["commands/scripts/*"]},
-        packages=find_packages(),
         include_package_data=True,
```

Comment:
this does not break anything with SFT / DPO CLI right?

Reply:
no, since they are python files in subdirectories they are already included:

```
  adding 'trl/commands/__init__.py'
  adding 'trl/commands/cli.py'
  adding 'trl/commands/cli_utils.py'
  adding 'trl/commands/scripts/chat.py'
  adding 'trl/commands/scripts/ddpo.py'
  adding 'trl/commands/scripts/dpo.py'
  adding 'trl/commands/scripts/kto.py'
  adding 'trl/commands/scripts/ppo.py'
  adding 'trl/commands/scripts/ppo_multi_adapter.py'
  adding 'trl/commands/scripts/reward_modeling.py'
  adding 'trl/commands/scripts/sft.py'
  adding 'trl/commands/scripts/config/default_chat_config.yaml'
 ```

Reply:
ok!

Reply:
ok fixed it